plugins/Toolbox/resources/qml/ToolboxConfirmUninstallResetDialog.qml

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2018 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.2
+import QtQuick.Controls 1.1
+import QtQuick.Controls.Styles 1.1
+import QtQuick.Layouts 1.1
+import QtQuick.Dialogs 1.1
+import QtQuick.Window 2.1
+
+import UM 1.3 as UM
+import Cura 1.0 as Cura
+
+
+UM.Dialog
+{
+    // This dialog asks the user whether he/she wants to open a project file as a project or import models.
+    id: base
+
+    title: catalog.i18nc("@title:window", "Confirm uninstall ") + toolbox.pluginToUninstall
+    width: 450 * screenScaleFactor
+    height: 50 * screenScaleFactor + dialogText.height + buttonBar.height
+
+    maximumWidth: 450 * screenScaleFactor
+    maximumHeight: 450 * screenScaleFactor
+    minimumWidth: 450 * screenScaleFactor
+    minimumHeight: 150 * screenScaleFactor
+
+    modality: UM.Application.platform == "linux" ? Qt.NonModal : Qt.WindowModal
+
+    Column
+    {
+        UM.I18nCatalog { id: catalog; name: "cura" }
+
+        anchors.fill: parent
+        anchors.leftMargin: Math.round(20 * screenScaleFactor)
+        anchors.rightMargin: Math.round(20 * screenScaleFactor)
+        anchors.topMargin: Math.round(10 * screenScaleFactor)
+        anchors.bottomMargin: Math.round(10 * screenScaleFactor)
```

Comment:
Personal preference but I think its a lot cleaner when this sort of thing is written as:

```
anchors
{
    fill: parent
    leftMargin: ... etc
}
```